canvas_modules/harness/src/styles/index.scss

```diff
@@ -26,8 +26,6 @@ $carbon--theme: $carbon--theme--g10;
 @import "carbon-components/scss/globals/scss/typography";
 
 
-@import "@elyra/canvas/src/index.scss";
```

Comment:
This was causing the reload to be very slow.  That's why the above styles in the test harness were updated.